Gemfile.lock

```diff
@@ -418,4 +422,4 @@ DEPENDENCIES
   whenever (= 0.9.4)
 
 BUNDLED WITH
-   1.11.2
+   1.12.5
```

Comment:
It doesn't match the version defined in puppet: https://github.com/alphagov/govuk-puppet/blob/master/modules/govuk_rbenv/manifests/all.pp#L64-L66 so it'll be a pain for those using the dev VM.